### PR TITLE
chore: small refactoring, typos and style fixes

### PR DESCRIPTION
**Pull request by TomerFi**
Signed-off-by: Tomer Figenblat <tfigenbl@redhat.com>

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -33,7 +34,7 @@ All you need is [VSCode][2] and [NodeJS][0].
 
 ### Launch Configurations
 
-- _Run Extension_ will run the extension in a separate _vsocde instance_.
```
Comment (ilan-pinto) [inline .github/CONTRIBUTING.md:36]: funny

File: README.md [hunk 2]
```diff
@@ -1,29 +1,33 @@
 # OCM VScode Extension
 
- "ocm-vscode-extension" helps creating OCM cluster applications. 
+The `ocm-vscode-extension` helps you quickly create OCM managed applications.
```
Comment (ilan-pinto) [inline README.md:3]: "you" is unneccery

File: src/extension.ts [hunk 3]
```diff
@@ -6,16 +6,12 @@ const extName = 'ocm-vscode-extension';
...
-		}
-	);
-}
+const cmdNewProjectDisposable = vscode.commands.registerCommand(
+	`${extName}.${cmdNewProjectName}`, () => newProject.create()
+);
 
 // EXPORTS
-export async function activate(context: vscode.ExtensionContext) {
+export function activate(context: vscode.ExtensionContext) {
```
Comment (ilan-pinto) [inline src/extension.ts:14]: why did you remove the async ?

Comment (TomerFi) [reply src/extension.ts:14]: It's not required at that point anymore.
it was a leftover from some older iteration when the disposable came for an async function that needed awaiting.

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (TomerFi) [reply .github/CONTRIBUTING.md:36]: it's your fault!
now that I know of _cspell_ I see all of my typos.
😁

File: README.md (hunk 2 above)
Comment (TomerFi) [reply README.md]: you are right, this is resolved in https://github.com/ilan-pinto/ocm-vscode-extension/pull/27/commits/1d31e2f0c8b27325518cbc050ea11ed26b57f18c
